src/NHibernate.Test/Async/NHSpecificTest/Logs/LogsFixture.cs

```diff
@@ -51,7 +51,7 @@ protected override string MappingsAssembly
 
 				await (s.GetAsync<Person>(1));//will execute some sql
 
-				var loggingEvent = spy.Events[0];
+				var loggingEvent = spy.GetWholeLog();
```

Comment:
@ngbrown, it seems you have forgotten to regen Async files.

Reply:
Not sure what you mean.  I run the `ShowBuildMenu` option H, and no files change.  Also, there's no documentation at https://github.com/maca88/AsyncGenerator, so what does it even do?

Reply:
Sorry, I was getting an error in the nuget restore, and didn't see that.

Reply:
#699 if you want to make a separate merge for that.

Reply:
[Contributing](https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md#regenerate-async-code) have been completed about that. But you now knows about it.

That is not a big change so I would not have required a new PR. I will see if Alexander will rather merge your PR first or just mine.